content/en/gitops.md

```diff
@@ -1,29 +1,22 @@
 ---
 title: GitOps
-status: Feedback Appreciated
+status: Completed
 category: Concept
 tags: ["methodology", "", ""]
 ---
 
-GitOps is a set of best practices based on [shared principles](https://opengitops.dev/), 
-applied to a workflow that depends on software agents that 
-enable automation to reconcile a declared system state or configuration in a git repository.
-These software agents and practices are used to execute a cohesive workflow that 
-leverages a source control system like Git as the “single source of truth” and 
-extends this practice to applications, infrastructure, and operational procedures.
+GitOps is a set of practices for managing software applications and infrastructure by continuously defining their desired states in a version control system such as Git.
```

Comment:
I wonder if we need to say "continuously" here now, given we describe driving updates in the next paragraph.

Reply:
From my perspective. Continuous is the hallmark of gitops and important to call out early. However the direction is wrong here. The states are defined, and then continuously reconciled to ensure actual state matches diesired state.